cmd/web.go

```diff
@@ -489,6 +489,20 @@ func runWeb(ctx *cli.Context) error {
 
 		m.Combo("/compare/*", repo.MustAllowPulls).Get(repo.CompareAndPullRequest).
 			Post(bindIgnErr(auth.CreateIssueForm{}), repo.CompareAndPullRequestPost)
+
```

Comment:
All the routes for creating/editing/uploading files to a repo and to make new branches